src/status_im/contexts/onboarding/events.cljs

```diff
@@ -115,14 +128,7 @@
                           :onboarding/navigated-to-enter-seed-phrase-from-screen
                           :screen/onboarding.create-profile)]
      (if (contains? (:profile/profiles-overview db) key-uid)
-       {:fx [[:effects.utils/show-confirmation
-              {:title               (i18n/label :t/multiaccount-exists-title)
-               :content             (i18n/label :t/multiaccount-exists-content)
-               :confirm-button-text (i18n/label :t/unlock)
-               :on-accept           (fn []
-                                      (re-frame/dispatch [:pop-to-root :screen/profile.profiles])
-                                      (re-frame/dispatch [:profile/profile-selected key-uid]))
-               :on-cancel           #(re-frame/dispatch [:pop-to-root :multiaccounts])}]]}
```

Comment:
We don't have a :multiaccounts root. Interestingly, pop-to-root didn't throw any errors, even with the incorrect root name, and it just worked because we also call dismiss-all-modals when invoking it.